### DISCUSSION TITLE
Repo'nun İngilizce olması daha mantıklı olabilir

### DISCUSSION THREAD

**Discussion by yafesdot** [evrimagaci/periodum#44] [General]
Projenin bu platformda daha çok kişiye ulaşabilmesi, daha kolay gelişebilmesi adına Issue şablonları ile ilgili açıklamaların ve bundan sonraki Issue'ler ile PR'ların Türkçe değil İngilizce olması daha mantıklı olabilir.

**Comment by cagrimertbakirci** [comment]
This is approved and the repository will from now on be English. 👍
